docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgis/postgis:16-3.5
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_DB=django
+    ports:
+      - "5432:5432"
```

Comment:
we could use a network and then use the internal address to avoid opening a port on the machine. But maybe that's something you want, for i.e. sql browser software